default.env

```diff
@@ -166,19 +159,16 @@ JWT_SECRET=
 DISTRIBUTED=
 # Authenticated execution client endpoint. This default uses the execution node container.
 EL_NODE=http://execution:8551
-# EL node for Lido Validator Ejector
-VE_EXECUTION_NODE_URL=http://execution:8551
 # Consensus client address. This could be comma-separated for Lighthouse or Teku VC clients, with failover,
 # or could just be a remote consensus client URL for "validator only" setups.
 # For Nimbus VC client, put the first beacon node here, and use VC_EXTRAS for additional.
 CL_NODE=http://consensus:5052
 # MEV-boost address. This would only be changed for Vouch setups
 MEV_NODE=http://mev-boost:18550
-# Beacon enpoint for OBOL node
-OBOL_NODE_BEACON_ENDPOINT=http://consensus:5052
-# Beacon enpoint for Lido Validator Ejector
-VE_BEACON_NODE_URL=http://consensus:5052
-LIDO_DV_EXIT_BEACON_NODE_URL=http://consensus:5052
+# Consensus client address for Charon and Lido Validator Ejector in Obol setup
+OBOL_CL_NODE=http://consensus:5052
+# Execution client address (RPC) for Lido Validator Ejector in Obol setup
+OBOL_EL_NODE=http://execution:8545
```

Comment:
Super. Please also increment ENV_VERSION so that ethd update will change .env to have these in. See upstream for current value of ENV_VERSION and go up by one.

Reply:
done